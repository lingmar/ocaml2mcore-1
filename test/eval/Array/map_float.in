let _ =
  let a = Array.make 5 0.0 in
  let aa = Array.map (fun x -> 1.0) a in
  Array.iteri (fun i x -> print_int i; print_float x; print_endline "") aa
