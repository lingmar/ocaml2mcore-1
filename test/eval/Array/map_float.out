01.0e-0
11.0e-0
21.0e-0
31.0e-0
41.0e-0
